.claude/skills/admin-ui/scripts/convention-check.sh

```
@@ -0,0 +1,191 @@
+# Helper Functions
+# =============================================================================
+
+warn() {
+    echo -e "${YELLOW}[CONVENTION WARNING]${NC} $1"
+    WARNINGS=$((WARNINGS + 1))
+}
+
+error() {
+    echo -e "${RED}[CONVENTION ERROR]${NC} $1"
+    exit 1
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)